### PR TITLE
Fix license header, typos and lints.

### PR DESCRIPTION
**Pull request by songy23**

### CODE REVIEW

File: receiver/prometheusreceiver/internal/internal_test.go [hunk 1]
```diff
@@ -17,7 +17,9 @@ package internal
 import (
 	"context"
 	"errors"
+
 	"github.com/open-telemetry/opentelemetry-service/consumer/consumerdata"
```
Comment (pjanotti) [inline receiver/prometheusreceiver/internal/internal_test.go]: nit: move this import to the group on the bottom

Comment (songy23) [reply receiver/prometheusreceiver/internal/internal_test.go]: Fixed.

**Comment by songy23** [comment]
> do you have the different tools/settings that you mentioned on issue #155? It will be good to put those in place too.

I'm not sure if there's a similar tool in open source. I'll take a look at the CI config of other Go github repos.

**Comment by songy23** [comment]
Thanks, commit messages fixed.

**Pull request merged**